### ISSUE TITLE
Accessibility features don't get updated on hot reload/restart

### ISSUE THREAD

**Issue by jonahwilliams** [flutter/flutter#20388]
This is probably similar to the textScaleFactor issue. To reproduce run on an android device and turn on disable animations in accessibility settings.  Then hot restart. Animations will play as normal.

**Comment by jonahwilliams** [comment]
These features ultimately come from the Android or iOS embedding, so they should be kept in sync. For example, If I have animations disabled on my android device a hot restart should not turn them back on
